colors/yowish.vim

```diff
@@ -51,7 +51,7 @@ call s:Hi('String'    , 'NONE' , s:color.lightGreen  , 'NONE')
 hi! link Character String
 " Default {{{1
 call s:Hi('ColorColumn'  , s:color.backgroundDark   , s:color.yellow          , 'bold')
-call s:Hi('Comment'      , 'NONE'                   , s:color.comment         , 'NONE')
+call s:Hi('Comment'      , 'NONE'                   , s:color.comment         , 'italic')
```

Comment:
A lot of people are not fan of italic comments (myself included :smile:), so IMO it should be optional via an option.

Better to add `g:yowish.comment_italic` in `autoload/yowish.vim` file, then check it here:

```vim
if g:yowish.comment_italic
    call s:Hi..... 'italic'
else
    call s:Hi..... 'NONE'
```